src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -109,6 +111,9 @@ const Body = BaseFormWithPolling.extend({
               // once the OV challenge succeeds,
               // triggers another polling right away without waiting for the next ongoing polling to be triggered
               // to make the authentication flow goes faster 
+              if (isEnhancedPollingEnabled === true) {
```

Comment:
See our conversations offline -- we should change this:
```suggestion
              if (deviceChallenge.enhancedPollingEnabled !== false) {
```